action/protocol/execution/evm/evm.go

```diff
@@ -316,6 +300,27 @@ func ExecuteContract(
 	return retval, receipt, nil
 }
 
+func depositeGas(ctx context.Context, sm protocol.StateManager, execution *action.Execution, consumedGas uint64, gasPrice *big.Int, hCtx HelperContext, fCtx protocol.FeatureCtx) (*action.TransactionLog, error) {
```

Comment:
this change isn't related to the intro of `WithHelperCtx`, which could be done in another pr.

Reply:
reverted